setup.py

```diff
@@ -10,7 +10,7 @@
     version='0.7.dev0',
```

Comment:
This should become a 1.0 as it is no longer Python 2 compatible.

Reply:
Sure. I'd recommend doing that outside of the PR though.